src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
```

Comment:
Please check your task on how your main method may look like. You should use Injector to get your Dao class.
https://github.com/EternalSadnes/JDBC-test-task-review#:~:text=In%20the%20main%20method%20call%20CRUD%20methods.%20It%20may%20look%20like%3A